server/controllers/tasks/deleteTask.js

```diff
@@ -1 +1,10 @@
-exports.deleteTask = (req, res) => res.send('single task delete');
+const deleteTask = require('./../../database/queries/deleteTask');
+
+exports.deleteTask = (request, response, next) => {
+  const { scrumId } = request.params;
+  deleteTask(scrumId)
+    .then((res) => {
+      response.send({ error: null, data: res.rows[0] });
```

Comment:
@aminalakhsham This all looks good! I don't think you need to return any data for the deleted item here (the frontend shouldn't need it), but 🤷‍♂️

Reply:
ty :D 
we agreed to return the data to test it.
your words are good but that what we agreed on it.

Reply:
Ah ok! sounds good